staging/src/k8s.io/apimachinery/pkg/util/validation/validation_test.go

```diff
@@ -154,6 +154,22 @@ func TestIsValidPortNum(t *testing.T) {
 	}
 }
 
+func TestIsInRange(t *testing.T) {
+	goodValues := [][]int{{1, 0, 10}, {5, 5, 20}, {25, 10, 25}}
+	for _, val := range goodValues {
+		if msgs := IsInRange(val[0], val[1], val[2]); len(msgs) != 0 {
+			t.Errorf("expected true for %d, got %v", val, msgs)
```

Comment:
`IsInRange` doesn't actually return a boolean, I think you expect `nil` here.
Similarly for `badValues` but with the reverse expectation.